neo4j/driver/row_reader_test.go

```diff
@@ -25,7 +25,13 @@ func TestBoltRowReader_Read(t *testing.T) {
 			},
 		}
 
-		rowReader := driver.NewBoltRowReader(mockBoltRows)
+		mockBoltConn := &internal.BoltConnMock{
+			CloseFunc: func() error {
```

Comment:
Little bit picky but you could make this nop close a var and reuse it to reduce the amount of code.

```
var closeNoErr = func() error {
    return nil
}
....
mockBoltConn := &internal.BoltConnMock{CloseFunc: closeNoErr}
```
Etc etc.

Reply:
good shout, will do